sprint 7/tema 3.js

```diff
@@ -0,0 +1,44 @@
+function dayOfTheWeek(weekDay) {
+  
+  let dayName = parseInt(weekDay);
+
+  switch(dayName) {
+    case 1:
+      return "Monday";
+
+    case 2:
+      return "Tuesday";
+
+    case 3:
+      return "Wednesday";
+
+    case 4:
+      return "Thursday";
+
+    case 5:
+      return "Friday";
+
+    case 6:
+      return "Saturday";
+
+    case 7:
+    case 0:
+      return "Sunday";
+
+    default:
+      return "You have to write a number between 0 and 7.";
```

Comment:
```suggestion
      throw "You have to write a number between 0 and 7.";
```
optional. Se poate arunca o exceptie aici